pennylane/operation.py

```diff
@@ -225,6 +225,13 @@ class Operator(abc.ABC):
             applicable `qml._current_context`. If no context is
             available, this argument is ignored.
     """
+    @staticmethod
+    def _matrix(*params):
+        """Matrix representation of the operator
+        in the computational basis.
```

Comment:
It might be worth explaining why there is a ``_matrix`` and a ``matrix`` method here, such as leaving an extra comment like ``This method allows `Operation` instances to return matrices for parametrized operations`` or something along those lines.

Reply:
done